packages/build/tests/secrets_scanning/snapshots/tests.js.md

```diff
@@ -1,4 +1,4 @@
-# Snapshot report for `tests/secrets_scanning/tests.js`
+# Snapshot report for `packages/build/tests/secrets_scanning/tests.js`
```

Comment:
Could this change be reverted please? I think it's an auto-generated change if you aren't in the expected folder when running the tests.